### ISSUE TITLE
windows version

### ISSUE THREAD

**Issue by zmztxzs** [OpenDigitalTwin-Dev/OpenDigitalTwin#7]
If there is windows version of OpenDigitalTwin code? we want to use if on windows platform.
Thanks!!

**Comment by fengsim** [comment]
Yes, we have already done the compilation for windows. Now we combined more than 50 open source projects to cover the technologies for CAD/CAE/CAM/CAI and this is the other open source project for CAX SDK (OpenCAXPlus). Maybe you could tell me what you need and we could send the SDK to you.
